webportal/lambda/deployment/deploy_lambda.py

```diff
@@ -4,6 +4,8 @@
 
 def deploy_backend(command_line_arguments):
     parsed_arguments = cli_parser.parse_arguments(command_line_arguments)
-    chalice_deploy_command = f'chalice deploy --stage {parsed_arguments.environment} --no-autogen-policy --profile sdc'
+    chalice_deploy_command = f'chalice deploy --stage {parsed_arguments.environment} --no-autogen-policy'
+    if not parsed_arguments.environment.startswith('ecs'):
+        chalice_deploy_command += ' --profile sdc'
```

Comment:
I was expecting `--profile ecs` or something like that. How are your creds being pulled?

Reply:
These are being run on an EC2 RHEL instance with a PowerUser profile. I'm just pretending I'm Jenkins running the Terraforms...